Tests/Kitura/TestResponse.swift

```diff
@@ -150,11 +150,41 @@ class TestResponse : XCTestCase {
                     "-----------------------------9051914041544843365972754266\r\n" +
                     "Content-Disposition: form-data; name=\"file1\"; filename=\"a.txt\"\r\n" +
                     "Content-Type: text/plain\r\n\r\n" +
-                    "Content of a.txt.\r\n\r\n" +
+                    "Content of a.txt.\r\n" +
```

Comment:
I do not believe there is any rule stating that you cannot have double cartage returns at the end like this, I do not see why it needs to be eliminated.